src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -16,9 +16,9 @@
  */
 class FormThemeNode extends \Twig_Node
 {
-    public function __construct(\Twig_Node $form, \Twig_Node $resources, $lineno, $tag = null)
+    public function __construct(\Twig_Node $form, \Twig_Node $resources, $only, $lineno, $tag = null)
```

Comment:
@HeahDude Will do! As I am currently on vacation, give me couple of weeks and I'll do it.